README.md

```diff
@@ -27,33 +27,52 @@ The demos in this repository run on the following platforms:
 
 The samples are meant to be used with a custom (prototype) engine using the master branch of the framework.
 
-0. Set up the Framework development environment: See [the wiki page](https://github.com/flutter/flutter/wiki/Setting-up-the-Framework-development-environment).
+0. Set up the Framework development environment: see [the wiki page](https://github.com/flutter/flutter/wiki/Setting-up-the-Framework-development-environment).
 
 1. In **the framework repo**, switch to the master branch:
 ```
 git checkout master
 ```
 
-2. Set up the Engine development environment: See [the wiki page](https://github.com/flutter/flutter/wiki/Setting-up-the-Engine-development-environment).
+2. Set up the Engine development environment: see [the wiki page](https://github.com/flutter/flutter/wiki/Setting-up-the-Engine-development-environment).
 
-3. Windows only: in **the engine repo**, apply this playground's patch:
-```
-git apply /path/to/playground/patches/001-Add-multi-view-Flutter-Windows-C++-APIs.patch
-```
+3. macOS only: in **the engine repo**, add the prototype remote and switch to the prototype branch:
 
-5. Build the custom engine: See [the wiki page](https://github.com/flutter/flutter/wiki/Compiling-the-engine#compiling-for-macos-or-linux).
+   ```bash
+   git remote add dkwingsmt https://github.com/dkwingsmt/engine/
+   git fetch dkwingsmt fffa6e19f5b71bedd55ac3a7fa6acfd85eda5983
+   git checkout fffa6e19f5b71bedd55ac3a7fa6acfd85eda5983
+   ```
 
-6. In **this repo**, update packages.
-```
-flutter pub get
-```
+4. macOS only: in **this repo**, edit pubspec.yaml, and add the following dependency override to use the custom `dart:ui` library:
```

Comment:
Why does macOS need to do this?

Reply:
Ah good point, that shouldn't be necessary any more.

Reply:
Removed!